### ISSUE TITLE
Latest version of pssh on PyPi

### ISSUE THREAD

**Issue by bhedayat** [lilydjwg/pssh#114]
Hello,

Is there a later version of pssh on PyPi https://pypi.org/project/pssh/. The last build was from 2012. 

I've had to deal with issues such as 
https://bugzilla.redhat.com/show_bug.cgi?id=1330231
https://inlovewithcode.wordpress.com/2017/10/06/pssh-python3-issue/
Which seemed to fix the problem, but I'd rather not fix it manually.

Thank you

**Comment by lilydjwg** [comment]
No. This is one of the many forks and I don't have access to the PyPI project.

**Comment by mgedmin** [comment]
There's a [process for taking over unmaintained PyPI projects](https://www.python.org/dev/peps/pep-0541/).  It takes a while (several months) while PyPI admins try to contact the original maintainer to verify that they're really non-responsive.

It would probably help if there was one common fork with all the fixes rather than multiple competing ones.

**Comment by lilydjwg** [comment]
The original maintainer actually responded, but he wished me to try to contact all other forks first, which I don't feel good to do: I'm not a very social person.

**Comment by jnooree** [comment]
I guess this is the only "survivor" of such forks. Shouldn't this repository be the official successor of the upstream?

The second least-to-the-last updated fork is [biffsocko/pssh3](https://github.com/biffsocko/pssh3), which seems not to be receiving updates after 2020.
